legend-depot-store-metrics/src/main/java/org/finos/legend/depot/store/metrics/services/QueryMetricsHandler.java

```diff
@@ -51,22 +60,30 @@ public Optional<VersionQueryMetric> getSummary(String groupId, String artifactId
     }
 
 
-    public List<VersionQueryMetric> getSummaryByProjectVersion()
+    public List<VersionQueryMetric> getSummaryByProjectVersion(boolean sorted)
```

Comment:
instead of pasing a boolean to sort or not , let the caller do the sort if needed